### ISSUE TITLE
Disable Auto Refresh: Please Open Power Pivot Window

### ISSUE THREAD

**Issue by mltytskr** [OlapPivotTableExtensions/OlapPivotTableExtensions#9]
I have a workbook with a lot of pivot tables (and too many calculated columns) and thought this would help out a lot. When trying to disable the auto refresh on the pivot table it tells me to open the Power Pivot window first or to restart Excel if that doesn't work. I have tried with the Power Pivot window open and closed and neither option works. I have restarted Excel numerous times (waited 20 minutes for the data to load back into memory when opening Power Pivot) yet I get the same error.

![image](https://user-images.githubusercontent.com/14055627/42528348-2b5fe442-8449-11e8-8941-0c29d2078fd5.png)

**Comment by furmangg** [comment]
I assume the PivotTable is connected to a PowerPivot data model? If you open the PowerPivot window manually you see your tables and measures?

**Comment by mltytskr** [comment]
Yes, it is a Power Query connected to Dynamics 365 and the queries are loaded into the Power Pivot data model then pulled into a pivot data.

**Comment by mltytskr** [comment]
I ended up needing to remove the add on. The Power Query window would load the ribbon but no queries (just a white window), and then my CPU would max at 100% for 10 minutes before I killed Excel. I uninstalled and Excel is working again.
